skills/src/main/java/cm/aptoide/skills/util/EskillsUri.kt

```diff
@@ -0,0 +1,67 @@
+package cm.aptoide.skills.util
+
+import com.google.gson.Gson
+import com.google.gson.reflect.TypeToken
+import java.math.BigDecimal
+
+data class EskillsUri(
```

Comment:
Not a big fan of how this data class is constructed, because as it stands we're parsing on the fly everytime we want a certain attribute. Also despite being a data class, the arguments are var when they should be val.

Personally I think this should be parsed only once in EskillsUriParser and this data class should contain the different parameters parsed from it (e.g. `data class EskillsUri(val userId: String?, val userName: String?....)`, although in this case, the data class should probably be renamed).

Reply:
refactored